pom.xml

```diff
@@ -429,7 +429,7 @@
       <versionx.org.wildfly.openssl.wildfly-openssl-macosx-x86_64>${version.org.wildfly.openssl.natives.macos_x86_64}</versionx.org.wildfly.openssl.wildfly-openssl-macosx-x86_64>
       <versionx.org.wildfly.openssl.wildfly-openssl-windows-x86_64>${version.org.wildfly.openssl.natives.windows_x86_64}</versionx.org.wildfly.openssl.wildfly-openssl-windows-x86_64>
       <versionx.org.wildfly.security.wildfly-elytron>${version.org.wildfly.elytron}</versionx.org.wildfly.security.wildfly-elytron>
-      <versionx.org.xerial>3.36.0.1</versionx.org.xerial>
+      <versionx.org.xerial>3.41.2.2</versionx.org.xerial>
```

Comment:
To provide context this https://github.com/infinispan/infinispan/commit/bc80814280f8e5e89e09e19ffd25bfb070e19f11